packages/components/src/call-out/index.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+	Card,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalHeading as Heading,
+	Icon,
+} from '@wordpress/components';
+import clsx from 'clsx';
+import { forwardRef } from 'react';
+import type { CallOutProps } from './types';
+import './styles.scss';
+
+function UnforwardedCallOut(
+	{ title, icon, illustration, actions, children }: CallOutProps,
+	ref: React.ForwardedRef< HTMLElement >
+) {
+	return (
+		<Card ref={ ref } className="call-out">
+			<HStack
+				ref={ ref }
```

Comment:
On the previous `ref` point, is it valid to be assigning it to both elements like this?

Reply:
Oops :)

> My instinct is that we could always accept the ref and always assign it to the top-most wrapping element

Without know the use case I think the most wrapping element is the most obvious candidate. It's most likely to be useful for something like collision detection.